.github/workflows/publish.yml

```diff
@@ -37,9 +37,9 @@ jobs:
 
     # DOCS
     - name: Set up Python 3.7 
-      uses: actions/setup-python@v2 
-      with: 
-        python-version: '3.7.5' 
+      uses: actions/setup-python@v4
+      with:
```

Comment:
I think this is not needed

Reply:
I'll revert this change back to v3.7.5. I figured I would update along with the others to stay consistent across all actions.